requirements_dev.txt

```diff
@@ -1,10 +1,11 @@
-pip==18.1
 bumpversion==0.5.3
-wheel==0.32.1
-watchdog==0.9.0
+coverage==4.5.1
 flake8==3.6.0
+pip==18.1
```

Comment:
Why is there a fixed version for `pip`?

Reply:
This is actually part of the cookie cutter library template. IMO if all packages were without version, it is more likely that errors will arise often.